src/express/commands/milestone.js

```diff
@@ -0,0 +1,400 @@
+/* eslint-disable dot-notation */
+import { promises } from 'fs-extra'
+import { loadDevnetConfig, splitToArray } from '../common/config-utils'
+import { timer } from '../common/time-utils'
+import { checkLatestMilestone } from './monitor';
+
+const {
+  runScpCommand,
+  runSshCommand,
+  maxRetries,
+  runSshCommandWithoutExit,
+  runSshCommandWithReturn
+} = require('../common/remote-worker')
+
+const milestoneLength = 64
+const queryTimer = (milestoneLength / 4) * 1000
+
+async function getLatestBlock(ip, number = "latest") {
+  const url = `http://${ip}:8545`
+  if (number != "latest") {
+    number = "0x" + Number(number).toString(16) // hexify
+  }
+
+  const response = await fetch(url, {
+    method: "POST",
+    headers: {"Content-Type": "application/json"},
+    body: JSON.stringify({
+      "jsonrpc":"2.0",
+      "id":1,
+      "method":"eth_getBlockByNumber",
+      "params":[number, false],
+    })
+  })
+
+  const responseJson = await response.json()
+  if (responseJson.result) {
+    return responseJson.result
+  } else {
+    console.log(`📍Error fetching block. number: ${number}, response: ${JSON.stringify(response)}`)
+  }
+
+  return undefined
+}
+
+async function removePeers(ip, peers) {
+  let tasks = []
+  for (let i = 0; i < peers.length; i++) {
+    let command = `~/go/bin/bor attach ~/.bor/data/bor.ipc --exec "admin.removePeer('${peers[i]}')"`
```

Comment:
shouldn't the paths be according to v0.3?
right now this may work but it should be made to new paths so that in future we don't need to revisit this

Reply:
Matic cli still uses the old paths and I saw other commands using the same so didn't change it. Maybe once all things are working we can create a global path variable for bor path and use it everywhere instead of hardcoding it.